### PR TITLE
add style

### PR DESCRIPTION
**Pull request by sajjadrajaee**
Add styling to the project:
Added styling for Book, Navbar and Form
I used best practice for HTML, CSS and JS
No linter errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable*/
```
Comment (HAMMAS-SALEEM) [inline src/App.js:1]: ### Your Summary
summary

### Descriptive Summary
descriptive-summary

- [ ] Your Summary is not descriptive. Please add a descriptive summary to make your pull request look professionl :blush:

File: src/App.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+/* eslint-disable */
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { Provider } from 'react-redux';
```
Comment (HAMMAS-SALEEM) [inline src/App.js:4]: ### Error
errors

- [ ] Your application is broken and not showing anything other than these errors please fix them to make your application work :+1:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle: :traffic_light: :rotating_light:

## Hey @sajjadrajaee :wave: 

**You have made a great effort on your project :blush: and I can see that but unfortunately I can not proceed with a code review of your pull request due to the following reasons :receipt:**

### Once you implement all the changes you should be ready to submit your project for a review :rocket: :rocket:

Cheers and Happy coding!👏👏👏

### Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. You can reach out to me on slack by searching my name (Hammas Bin Saleem) for any explanation.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @sajjadrajaee,

Good job working on the changes asked by the previous reviewer.  💪

Your project is complete!  You did well on the following: 💪💪💪

✅ You used Gitflow correctly
✅ No linter error
✅ README is ok
✅ Good descriptive PR
✅ App works fine and the app is designed as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by sajjadrajaee** [comment]
Thank you for your help and review. 👍

**Pull request merged**